doc/howto/move_instances.md

```diff
@@ -7,7 +7,9 @@ To move an instance from one LXD server to another, use the `lxc move` command:
 
 ```{note}
 When moving a container, you must stop it first.
-See {ref}`live-migration` for more information.
+See {ref}`live-migration-containers` for more information.
+
+When moving a virtual machine, you must either enable {ref}`live-migration-vms` or stop it first.
```

Comment:
Maybe "setup" here rather than "enable"? Otherwise it sounds to me like its a single setting one has to enable.

Reply:
`` {ref}`live-migration-vms` `` resolves to a link with the text "Live migration for virtual machines", so I don't think anybody will confuse it with a single setting. And that section is about how to enable it, so I feel "enable" (make it possible) fits better here than "set up" (bring into use).

Reply:
OK makes sense